canonical/canonical.com
# Fix missing timezone errors

## Done
- Some countries are missing timezone values, but they have country aliases that has the same timezone values
- Check if country has missing timezone and get the timezone from their respective aliases
- Added fallback country (GB) for invalid timezone queries

## QA

- Check out demo
- For valid timezones, check that the country code and country shown is as expected. 
   Query:
  - `/user-country-tz.json?tz=Asia/Kuala_Lumpur`
  - `/user-country-tz.json?tz=America/Mendoza`
- For invalid timezones, the fallback value should be **{ "country": "United Kingdom",  "country_code": "GB" }**.
   Query:
  - `/user-country-tz.json?tz=Etc/GMT-8`
  - `/user-country-tz.json?tz=test`


## Issue / Card

Fixes [WD-13251](https://warthogs.atlassian.net/browse/WD-13251)

## Screenshots

[if relevant, include a screenshot]
